__tests__/app.test.js

```diff
@@ -1,23 +1,38 @@
-const app = require('../app')
-const request = require('supertest')
-const db = require('../db/connection')
-const seed = require('../db/seeds/seed')
-const testData = require('../db/data/test-data')
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
 
 beforeEach(() => {
-    return seed(testData)
-})
+  return seed(testData);
+});
 afterAll(() => {
-    db.end()
-})
+  db.end();
+});
 
-describe('GET /api/categories', () => { 
-    test('200, should return all the categories with the relevant keys', () => {
-        return request(app)
-        .get("/api/categories")
-        .expect(200)
-        .then((res) => {
-            console.log(res.body.categories);
-        })
-    });
-})
\ No newline at end of file
+describe('GET /api/categories', () => {
+  test('200: should return an array of all the categories objects', () => {
+    return request(app)
+      .get('/api/categories')
+      .expect(200)
+      .then(({body}) => {
+        expect(body.categories).toHaveLength(4);
+        body.categories.forEach(category => {
+          expect(typeof category).toBe('object');
+          expect(typeof category.name).toBe('string');
+          expect(typeof category.description).toBe('string');
```

Comment:
This is good. Good to ensure that we're getting the correct data type coming through